common/alert/reports/a11y-1.json

```
@@ -0,0 +1,908 @@
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.2.2",
+      "earl:test": "WCAG21:captions-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.2.3",
+      "earl:test": "WCAG21:audio-description-or-media-alternative-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "dct:description": "",
+      "earl:result": [],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.2.4",
+      "earl:test": "WCAG21:captions-live",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.2.5",
+      "earl:test": "WCAG21:captions-live",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.3.1",
+      "earl:test": "WCAG21:info-and-relationships",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:failed",
+      "dct:description": "It's failing because I believe when we are using div, there is no alternative way. But it's passing for section because of heading; Ref:https://www.w3.org/TR/wai-aria-1.2/#strong",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G115",
+            "earl:outcome": "earl:failed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/H49",
+            "earl:outcome": "earl:failed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/H42",
+            "earl:outcome": "earl:failed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:failures/F43",
+            "earl:outcome": "earl:failed",
+            "earl:subject": "_:subject"
+        },
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.3.2",
+      "earl:test": "WCAG21:meaningful-sequence",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G57",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
```

To fix the CI errors about JSON validation
```suggestion
      "earl:result": [
        {
            "earl:test": "WCAG2xTech:general/G57",
            "earl:outcome": "earl:passed",
            "earl:subject": "_:subject"
        }
      ],
      "earl:mode": "earl:manual",
      "@type": ["earl:TestResult", "earl:Assertion"]
```